index.html

```diff
@@ -329,6 +326,42 @@ <h2 class="top-heading">About &nbsp; &nbsp; &nbsp; &nbsp; Myself</h2>
           <img class="sep1" src="./images/Separator Bottom.png" alt="line" />
         </div>
       </section>
+      <section class="Contact-me">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/{form_id}" methode="post">
+          <input
+            type="text"
+            name="user name"
+            placeholder="Girma Anwaar"
+            required
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="email"
+            placeholder="girma@gmail.com"
+            required
+            placeholder="user name"
+            required
+            maxlength="30"
```

Comment:
- [ ] Email should not have this `maxlength=30` validation because you can't be strict on how users' email should look. Please remove it 😉